src/authManager.ts

```diff
@@ -116,7 +116,7 @@ export default class AuthManager {
         const responseJson = await response.json();
 
         this.jwt = responseJson.jwt;
-        this.active = responseJson.subscription.isActive;
+        this.active = responseJson.subscription && true; // set to true as long as we have subscription
```

Comment:
If we want to convert any object to boolean we can do this:
```suggestion
        this.active = !!responseJson.subscription; // set to true as long as we have subscription
```